docs/reference/modules/discovery/quorums.asciidoc

```diff
@@ -0,0 +1,181 @@
+will be used and is controlled by the following setting.
+
+`cluster.auto_shrink_voting_configuration`::
+
+    Defaults to `true`, meaning that the voting configuration will automatically
+    shrink, shedding departed nodes, as long as it still contains at least 3
+    nodes.  If set to `false`, the voting configuration never automatically
+    shrinks; departed nodes must be removed manually using the
+    <<modules-discovery-adding-removing-nodes,voting configuration exclusions API>>.
+
+NOTE: If `cluster.auto_shrink_voting_configuration` is set to `true`, the
+recommended and default setting, and there are at least three master-eligible
+nodes in the cluster, then Elasticsearch remains capable of processing
+cluster-state updates as long as all but one of its master-eligible nodes are
+healthy.
+
+There are situations in which Elasticsearch might tolerate the loss of multiple
+nodes, but this is not guaranteed under all sequences of failures. If this
+setting is set to `false` then departed nodes must be removed from the voting
+configuration manually, using the
+<<modules-discovery-adding-removing-nodes,voting exclusions API>>, to achieve
+the desired level of resilience.
+
+Note that Elasticsearch will not suffer from a "split-brain" inconsistency
+however it is configured. This setting only affects its availability in the
+event of the failure of some of its nodes, and the administrative tasks that
+must be performed as nodes join and leave the cluster.
+
+[float]
+==== Even numbers of master-eligible nodes
+
+There should normally be an odd number of master-eligible nodes in a cluster.
+If there is an even number then Elasticsearch will leave one of them out of the
+voting configuration to ensure that it has an odd size. This does not decrease
+the failure-tolerance of the cluster, and in fact improves it slightly: if the
+cluster is partitioned into two even halves then one of the halves will contain
+a majority of the voting configuration and will be able to keep operating,
+whereas if all of the master-eligible nodes' votes were counted then neither
+side could make any progress in this situation.
+
+For instance if there are four master-eligible nodes in the cluster and the
+voting configuration contained all of them then any quorum-based decision would
+require votes from at least three of them, which means that the cluster can only
+tolerate the loss of a single master-eligible node. If this cluster were split
+into two equal halves then neither half would contain three master-eligible
+nodes so would not be able to make any progress. However if the voting
+configuration contains only three of the four master-eligible nodes then the
+cluster is still only fully tolerant to the loss of one node, but quorum-based
+decisions require votes from two of the three voting nodes. In the event of an
+even split, one half will contain two of the three voting nodes so will remain
+available.
```

Comment:
It's not clear what "will remain available".  The cluster?  Decision-making progress?

Reply:
added "that half" in f24c1d9dc52.